### PR TITLE
Jahia link update

### PR DESCRIPTION
**Pull request by sean-lockwood**
**DO NOT MAKE THESE CHANGES LIVE UNTIL THE NEW WEBSITE IS UP AND RUNNING!**

The Zope --> Jahia transition is currently scheduled for Monday, June 17, 2019. I updated links, but most cannot be tested until after the new site goes live.

Some additional links may need to be changed.  All will need to be tested before deployment -- I suggest running sphinx's `make linkcheck`.

FTP link changes have been removed from this PR.

~Here are a few links I haven't updated, but will probably need to be modified:~
 - ~http://www.stsci.edu/hst/observatory/crds/SIfileInfo/pysynphottables/index_phoenix_models_html~
 - ~http://www.stsci.edu/hst/observatory/crds/documents/TIR-CDBS-2012-01.pdf~
 - ~http://www.stsci.edu/hst/HST_overview/documents/synphot/hst_synphotTOC.html~
 - ~https://aeon.stsci.edu/ssb/trac/astrolib/ticket/218~

~Things I updated but are worthy of further consideration:~
 - ~http://www.sdss.org/dr3/instruments/imager/#filters --> dr7~
 - ~http://www.sdss.org/dr3/algorithms/fluxcal.html     --> dr7~

Closing PR #113.

### CODE REVIEW

File: doc/source/appendixb.rst [hunk 1]
```diff
@@ -1233,7 +1233,7 @@ large extended sources (larger than about 80 pixels in size)
 because the extended IR scattering wings in these bands,
 which do not affect the photometry of point sources, begin to be
 included. The modified curves are shown in an
-`updated SDSS system response plot <http://www.sdss.org/dr3/instruments/imager/#filters>`_.
+`updated SDSS system response plot <http://classic.sdss.org/dr7/instruments/imager/#filters>`_.
```
Comment (pllim) [inline doc/source/appendixb.rst]: This is not related to Zope. And I don't really know if updating this link reflects what is actually in CDBS. You have to confirm with RedCat first. Please revert `dr7` change from this PR.

**Comment by pllim** [comment]
linkcheck is a good idea, though it is prone to timeout errors. Feel free to improve the CI if you want, as I don't have much time devoted to this package.

File: doc/source/appendixb.rst (hunk 1 above)
Comment (sean-lockwood) [reply doc/source/appendixb.rst]: Reverted `dr7` in force-push below.

**Comment by sean-lockwood** [comment]
Will do.

I was thinking of linkcheck mainly as a manual check during this transition.

**Comment by pllim** [comment]
@sean-lockwood , will http://www.stsci.edu/hst/observatory/crds/SIfileInfo/pysynphottables/index_phoenix_models_html be affected? I see that it is not changed in your PR.

**Comment by sean-lockwood** [comment]
I believe it will be.  I just couldn't find the equivalent page on the new site.

**Comment by pllim** [comment]
@sean-lockwood , did you approach RedCat about the remaining http://www.stsci.edu/hst/observatory/crds links?

http://www.stsci.edu/hst/HST_overview -- Is this Zope? I don't know. Maybe @dancoe knows.

https://aeon.stsci.edu/ssb/trac/astrolib -- This is still online and not affected by Zope, right? We can leave it alone for now.

**Comment by pllim** [comment]
And what about http://www.stsci.edu/hst/acs/analysis/zeropoints ; is this going anywhere?

**Comment by pllim** [comment]
Oh sorry. I missed that in the diff. Nvm

**Comment by sean-lockwood** [comment]
I've notified Rossy about the two remaining links:
- http://www.stsci.edu/hst/observatory/crds/documents/TIR-CDBS-2012-01.pdf
- http://www.stsci.edu/hst/HST_overview/documents/synphot/hst_synphotTOC.html

**Comment by pllim** [comment]
@sean-lockwood , what about http://www.stsci.edu/hst/observatory/crds/SIfileInfo/pysynphottables/index_phoenix_models_html ? It seems to redirect to a generic landing page.

**Comment by pllim** [comment]
Oh, never mind. Just noticed your latest commit. Thanks!

**Comment by sean-lockwood** [comment]
Pushed link to http://www.stsci.edu/files/live/sites/www/files/home/hst/documentation/_documents/hst_synphot.pdf

**Comment by pllim** [comment]
Only one left. :smile:

**Comment by sean-lockwood** [comment]
Last link has been pushed.  Phew!

**Comment by pllim** [comment]
So, is this ready for merge, @sean-lockwood ?

**Comment by sean-lockwood** [comment]
Good to go!

**Comment by pllim** [comment]
Thank you very much!
